sdk/anomalydetector/ai-anomaly-detector/README.md

```diff
@@ -87,7 +91,7 @@ const client = new AnomalyDetectorClient("<endpoint>", new DefaultAzureCredentia
 
 ## Examples
 
-Coming soon (#10865)
+Samples can be found [here](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/anomalydetector/ai-anomaly-detector/samples)
```

Comment:
Typically in this section we add code snippets for champion scenarios. We are not doing that here?

Reply:
I couldn't find a doc site with the information but added more details on how to get it in the portal

Reply:
Created https://github.com/Azure/azure-sdk-for-js/issues/11328 for adding the snippets